sirepo/template/raydata.py

```diff
@@ -173,19 +179,17 @@ def _metadata(data):
     return res
 
 
-def _scan_info(scan_uuid, metadata=None):
+def _scan_info(scan_uuid, selected_columns, metadata=None):
```

Comment:
`selected_columns` is a new argument and breaks other callers that need to be fixed.

Reply:
e64969d8a3a9303ebcf97566971db0411515341f